src/Component/AutoMapper/Extractor/MapToContextReadInfoExtractorDecorator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jane\Component\AutoMapper\Extractor;
+
+use Jane\Component\AutoMapper\Attribute\MapToContext;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\PropertyReadInfo;
+use Symfony\Component\PropertyInfo\PropertyReadInfoExtractorInterface;
+
+final class MapToContextReadInfoExtractorDecorator implements PropertyReadInfoExtractorInterface
```

Comment:
`ReflectionExtractor::getReadInfo()` does not allow [a getter with mandatory attributes](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php#L258).

Let's decorate it and check if all parameters have the attribute in the case where the read info was a property not public. I think it is the unique valid case where you could use the feature in this PR: you have a private property and you want the automapper to use a public getter